src/index.html

```diff
@@ -7,6 +7,308 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
```

Comment:
```suggestion
     <header class="header">

      <div class="header__logo"></div>
```
Відступ між ними не потрібен

Reply:
також використовуй для лого тег img